Code/GraphMol/FileParsers/CDXMLParser.cpp

```diff
@@ -116,6 +117,25 @@ std::vector<T> to_vec(const std::string &s) {
   return n;
 }
 
+void scaleBonds(const ROMol &mol, Conformer &conf, double targetBondLength, double bondLength) {
+  double avg_bond_length = 0.0;
+  if(bondLength == -1) {
```

Comment:
rather than doing equality testing on a double, how about:
```suggestion
  if(bondLength < 0) {
```